apps/nextjs-website/src/components/organisms/Auth/EditPasswordForm.tsx

```diff
@@ -0,0 +1,138 @@
+import { ButtonNaked } from '@pagopa/mui-italia';
+import { Stack, Typography } from '@mui/material';
+import { useCallback, useState } from 'react';
+
+import { translations } from '@/_contents/translations';
+import { PasswordTextField } from './PasswordTextField';
+import { useTranslations } from 'next-intl';
+import { passwordMatcher } from '@/helpers/auth.helpers';
+
+type EditPasswordFormProps = {
+  onSave: (oldPassword: string, newPassword: string) => Promise<void>;
+  // eslint-disable-next-line functional/no-return-void
+  onCancel: () => void;
+};
+
+type Passwords = {
+  current_password: string;
+  new_password: string;
+  password_confirm: string;
+};
+
+export const EditPasswordForm = ({
+  onSave,
+  onCancel,
+}: EditPasswordFormProps) => {
+  const { auth } = translations;
+  const t = useTranslations('profile');
+  const [errors, setErrors] = useState<Partial<Passwords>>({});
+  const [passwords, setPasswords] = useState<Passwords>({
+    current_password: '',
+    new_password: '',
+    password_confirm: '',
+  });
+
+  const validateForm = useCallback(() => {
+    const { current_password, new_password, password_confirm } = passwords;
+    // eslint-disable-next-line functional/no-let
+    let err = {};
+
+    if (!current_password) {
+      err = { current_password: t('renewPassword.requiredCurrentPassword') };
+    }
+
+    if (!passwordMatcher.test(new_password)) {
+      err = { ...err, new_password: auth.signUp.passwordPolicy };
+    } else if (new_password !== password_confirm) {
+      err = { ...err, new_password: t('renewPassword.passwordsNotMatch') };
+    }
+
+    setErrors(err);
+    const hasErrors = Object.keys(err).length > 0;
+    return !hasErrors;
+  }, [passwords, auth.signUp.passwordPolicy, t]);
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const {
+      target: { value, name },
+    } = event;
+
+    setPasswords((prev) => ({ ...prev, [name]: value }));
+  };
+
+  const handleSave = () => {
+    if (!validateForm()) return;
+    onSave(passwords.current_password, passwords.new_password).catch(
+      (error) => {
+        if (error.__type === 'NotAuthorizedException') {
+          setErrors({ current_password: t('renewPassword.wrongPassword') });
+        } else {
```

Comment:
I think you forgot to show the wrong current password error:
Figma
![image](https://github.com/pagopa/developer-portal/assets/39835990/041d0dab-6340-484d-bc08-7f853589b679)
Your implementation
![image](https://github.com/pagopa/developer-portal/assets/39835990/3f0ca9a4-8953-4991-aeda-7d8a9cbcfc1a)

Reply:
As you can see in the previous screen capture, there's a problem with the title "cambia password" margin-bottom